test/es-module/test-esm-import-flag.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
+
+require('../common');
+const fixtures = require('../common/fixtures');
+const assert = require('assert');
+const { spawnSync } = require('child_process');
+
+{
+  // known limitation:
+  // imports should not be imported when evaluating text
+  const { status, stderr, stdout } = spawnSync(
+    process.execPath,
+    [
+      '--import',
+      fixtures.path('es-modules', 'mjs-file.mjs'),
```

Comment:
Surely `--import` should accept URLs rather than path, so it can be used with HTTPS URLs in the future.
```suggestion
      fixtures.fileURL('es-modules', 'mjs-file.mjs'),
```